### PR TITLE
Win10; Improved readability of code and comments

### PR DESCRIPTION
**Pull request by markuskramerIgitt**
Improvements:
 - Use Get-WindowsOptionalFeature to detect .Net Framework
 - Tested with 2018.3.3
 - Account for Build tools 2015 bugfix upgrade
 - Requirements raised to Windows 7/Server 2012

### CODE REVIEW

**Comment by markuskramerIgitt** [comment]
@twangboy could you please consider this pull request?

File: README.md [hunk 1]
```diff
@@ -88,16 +112,27 @@ properties, and the current value of those properties:
...
 
+### Naming conventions
 
+### For WiX
 
+Prefix  | Example                 | Meaning
+------- | ----------------------- | -------
+`IMCA_` | `IMCA_NukeConf`         | Immediate custom action
+`DECA_` | `DECA_SetMaster`        | Deferred custom action
+`CADH_` | `CADH_SetMaster`        | Custom action data helper for DECA_
```
Comment (twangboy) [inline README.md]: Should be CADH_ at the end, right?

File: README.md [hunk 2]
```diff
@@ -124,12 +159,13 @@ are required:
...
 - The new dialog file.
-- WixUI_Minion.wxs: &lt;Publish /&gt; entries hooking up the dialog buttons to other dialogs. 
+- WixUI_Minion.wxs: &lt;Publish /&gt; entries hooking up the dialog buttons to other dialogs.
   Other dialogs will also have to be adjusted to maintain correct sequencing.
 - MinionMSI.wixproj: The new dialog must be added as a &lt;Compile /&gt; item to be included in the build.
 
+### Other
 
-
+Disutils contain bin/Lib/distutils/command/bdist_msi.py, which probably does not work.
```
Comment (twangboy) [inline README.md]: Distutils?

File: log/build-screen-log.txt [hunk 3]
```diff
@@ -0,0 +1,122 @@
+
```
Comment (twangboy) [inline log/build-screen-log.txt]: Does the log file need to be a part of this repo?

File: README.md (hunk 2 above)
Comment (markuskramerIgitt) [reply README.md]: Thank, you corrected.

File: log/build-screen-log.txt (hunk 3 above)
Comment (markuskramerIgitt) [reply log/build-screen-log.txt]: The log files under revision control allows for an easy comparison of the current one with the "last good version"

File: README.md (hunk 1 above)
Comment (markuskramerIgitt) [reply README.md]: No, this is correct: the Deferred custom action (DECA_) has a related Custom action data helper.

**Comment by markuskramerIgitt** [comment]
Hi @waynew, on May31, what do you mean with 

>  GitHub only showed me one revision -_-

File: log/build-screen-log.txt (hunk 3 above)
Comment (markuskramerIgitt) [reply log/build-screen-log.txt]: Removed log, no longer deemed useful

File: README.md (hunk 1 above)
Comment (markuskramerIgitt) [reply README.md]: Does this answer your question?

Comment (twangboy) [reply README.md]: Yes

**Comment by waynew** [comment]
@markuskramerIgitt I reviewed an out-of-date revision 😝 I thought I was viewing the whole thing, but I wasn't.

**Comment by markuskramerIgitt** [comment]
@waynew, I see. Thank you

**Pull request merged**